react/week3/my_first_app/src/index.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+//import App from './App';
+import Header from './components/Header';
+import Counter from './components/Counter';
+import * as serviceWorker from './serviceWorker';
+import "react-datepicker/dist/react-datepicker.css";
+
+const root = document.getElementById('root');
+
+function fetchTodo() {
+  const URL = `https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw`
+  return fetch(URL)
+    .then(response => response.json())
+}
+
+class TodoListItem extends React.Component {
+
+  render() {
+    const isEdit = this.props.todo.canEdit
+    const isUpdate = this.props.todo.canUpdate
```

Comment:
I would consider how you're naming this, it reads a bit ambiguous. Are you saying `isEdit` as in it has been edited or `isEdit` as in this can be edited? I would use the name `isEditable` or `canEdit` respectively.